service/src/main/java/tds/assessment/services/impl/AssessmentServiceImpl.java

```diff
@@ -22,19 +25,17 @@
 @Service
 class AssessmentServiceImpl implements AssessmentService {
     private final AssessmentQueryRepository assessmentQueryRepository;
-    private final ItemQueryRepository itemQueryRepository;
-    private final FormQueryRepository formQueryRepository;
+    private final AssessmentAssembler assessmentAssembler;
     private final StrandQueryRepository strandQueryRepository;
 
     @Autowired
     public AssessmentServiceImpl(AssessmentQueryRepository assessmentQueryRepository,
                                  ItemQueryRepository itemQueryRepository,
                                  FormQueryRepository formQueryRepository,
                                  StrandQueryRepository strandQueryRepository) {
-        this.formQueryRepository = formQueryRepository;
         this.assessmentQueryRepository = assessmentQueryRepository;
-        this.itemQueryRepository = itemQueryRepository;
         this.strandQueryRepository = strandQueryRepository;
+        this.assessmentAssembler = new AssessmentAssembler(itemQueryRepository, formQueryRepository);
```

Comment:
I don't love this part.  It seems like a smell to me to pass in an `ItemQueryRepository` and `FormQueryRepository` implementation to the `AssessmentServiceImpl`, only to "pay it forward" to the `AssessmentAssembler`.  I thought about making the `AssessmentAssembler` its own legit service (complete w/interface), but that seemed a bit like overkill - how many `AssessmentAssembler` implementations are we likely to have?  So this is what I went with, but I'm certainly willing to make a change if someone has any thoughts.